fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1634,6 +1636,20 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
+    updateMinimums() {
+        const ssm = this.shouldTransmitMinimums() ? Arinc429Word.SignStatusMatrix.NormalOperation : Arinc429Word.SignStatusMatrix.NoComputedData;
+
+        this.setBnrArincSimVar('MDA', this.perfApprMDA ? this.perfApprMDA : 0, ssm, 17, 131072, 0);
+        this.setBnrArincSimVar('DH', this.perfApprDH ? this.perfApprDH : 0, ssm, 16, 8192, 0);
+    }
+
+    shouldTransmitMinimums() {
+        //Only transmit when in Cruise or later phase and within 200NM of destination
+        return (this.flightPhaseManager.phase != FmgcFlightPhases.PREFLIGHT
+            && this.flightPhaseManager.phase != FmgcFlightPhases.TAKEOFF
+            && this.flightPlanManager.getDistanceToDestination() <= 200);
```

Comment:
Our latest thought was this might be the same as the approach message on the ND, which has the following logic:
```suggestion
        return this.flightPhaseManager.phase > FmgcFlightPhase.Cruise || (this.flightPhaseManager.phase === FmgcFlightPhase.Cruise && this.flightPlanManager.getDistanceToDestination(FlightPlans.Active) < 250);
```

Reply:
Yearh makes sense, I will keep it as a small helper function, that way it should be possible to rename it and move it to somewhere appropriate in the typescript section.